composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "silverstripe/tx-translator",
+    "description": "Silverstripe TX Translator",
+    "type": "silverstripe-vendormodule",
+    "authors": [
+        {
+            "name": "SilverStripe",
+            "homepage": "http://silverstripe.com"
+        },
+        {
+            "name": "The SilverStripe Community",
+            "homepage": "http://silverstripe.org"
+        }
+    ],
+    "require": {
+        "php": "^7.4 || ^8.1",
+        "symfony/process": "^4 || ^5 || ^6",
+        "symfony/yaml": "^4 || ^5 || ^6",
```

Comment:
process and yaml aren't technically able to install alongside framework with `^5`, though `^4 || ^6` just looks weird and it's easier to just have `^4 || ^5 || ^6` so people don't get confused by this in the future